src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -27,17 +27,16 @@ public class AutoScoreAndBalance extends SequentialCommandGroup {
 
   public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive, ScoringLocationUtil scoringLocationUtil) {
     addCommands(
-      new InstantCommand(()->scoringLocationUtil.toggleMiddleGrid()), //is this needed??
       new InstantCommand(() -> scoringLocationUtil.setScoreCol(ScoreCol.RIGHT)),
       new InstantCommand(() -> scoringLocationUtil.setScoreHeight(ScoreHeight.HIGH)),
```

Comment:
We actually don't need the `scoringLocationUtil` *at all*. Like Basel said, it's pretty much only used for driving to a specific location. The arm doesn't rotate, so it isn't needed for positioning the arm or anything. And this auto has the bot already positioned at the right place, so we aren't driving anywhere.

Reply:
> Since this auto has the robot already at the scoring location, I think we can just totally ignore the scoring util for this auto. We only really need it for automatically driving to a location, which we don't need to do for this one

From Basel's response to my comment above

Reply:
Whoops ok. Will fix.